airbyte-integrations/connectors/source-sendgrid/manifest.yaml

```diff
@@ -637,11 +637,13 @@ definitions:
             api_token: "{{ config['api_key'] }}"
         download_requester:
           type: HttpRequester
-          path: "{{stream_slice.extra_fields['url']}}"
+          path: "{{download_target}}"
```

Comment:
With the latest CDK update applied to this source, we can also move from using `path` to set the `{{ download_target }}` directly in `url_base` and remove the `path` property from the `download_requester`.

Reply:
@bazarnov So the url_base should be {{download_target}}?